eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerTest.java

```diff
@@ -0,0 +1,114 @@
+package uk.ac.ebi.eva.server.ws;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.path.json.JsonPath;
+import com.jayway.restassured.response.Response;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Tom Smith
+ */
+public class VariantWSServerTest {
```

Comment:
Just to confirm, this are regression tests, right? I mean, tests for checking that the previous version works, so that when we use the new repository, the tests continue working.